### PR TITLE
Fix namespace

### PR DESCRIPTION
**Pull request by cnight**
Proper pull request for a couple namespace fixes.

### CODE REVIEW

**Comment by LBRapid** [comment]
Hey I'll merge this when I can. We are running SpreeConf right now and things are a bit hectic. I also only have my phone today. Cheers.

File: app/controllers/spree/checkout_controller_decorator.rb [hunk 1]
```diff
@@ -252,7 +252,7 @@ def order_opts(order, payment_method, stage)
         credits_total = credits.map {|i| i[:amount] * i[:quantity] }.sum
       end
 
-      opts = { #:return_url        => request.protocol + request.host_with_port + "/spree/orders/#{order.number}/checkout/paypal_confirm?payment_method_id=#{payment_method}",
+      opts = { #:return_url        => request.protocol + request.host_with_port + "/orders/#{order.number}/checkout/paypal_confirm?payment_method_id=#{payment_method}",
```
Comment (radar) [inline app/controllers/spree/checkout_controller_decorator.rb]: This would break if Spree was mounted at another location, other than root.

Comment (cnight) [inline app/controllers/spree/checkout_controller_decorator.rb]: Good point Ryan, I will fix this today.  Is there a way to update my
current pull request or do I need to issue a new one?

On Thu, Feb 16, 2012 at 9:17 AM, Ryan Bigg
reply@reply.github.com
wrote:

Comment (radar) [inline app/controllers/spree/checkout_controller_decorator.rb]: You can update it by pushing to your fix_namespace branch.

Comment (cnight) [inline app/controllers/spree/checkout_controller_decorator.rb]: Thanks, already got it pushed!

On Thu, Feb 16, 2012 at 11:51 AM, Ryan Bigg
reply@reply.github.com
wrote:

**Comment by LBRapid** [comment]
merged to master

**Pull request closed without merging**